code/game/objects/items/weapons/switchtool.dm

```diff
@@ -7,6 +7,9 @@
 	siemens_coefficient = 1
 	force = 3
 	w_class = W_CLASS_SMALL
+	var/no_remove = 0
```

Comment:
There's not really a need for this variable. Just set `removing_item` to null if you don't want things to be removable.